pkg/resources/etcd/testdata/etcd-command-with-launcher.golden.sh

```diff
@@ -0,0 +1 @@
+/opt/bin/etcd-launcher -namespace $(NAMESPACE) -etcd-cluster-size $(ETCD_CLUSTER_SIZE) -pod-name $(POD_NAME) -pod-ip $(POD_IP) -api-version $(ETCDCTL_API) -token $(TOKEN)
```

Comment:
Is this used somewhere?

Reply:
Yes, it's used by the test to compare the output of the command function.

Reply:
Ok, I missed it https://github.com/kubermatic/kubermatic/blob/2e30c15e5123cfa3d381ccaaa5f9641f561f3111/pkg/resources/etcd/statefulset_test.go#L76